docs/Concepts.md

```diff
@@ -105,7 +105,7 @@ For example:
 * A task's input could be a github source which contains your application code.
 * A task's output can be your application container image which can be then deployed in a cluster.
 
-Read more on PipelineResources and their types [here](./docs/pipeline-resources.md)
+Read more on PipelineResources and their types [here](./pipeline-resources.md)
 
```

Comment:
this link is still broken because the resources file is deleted, should be pointing to `using.md`

Reply:
oh good point :sweat_smile:

Reply:
Updated :wink: